ktor-client/ktor-client-curl/src/io/ktor/client/engine/curl/CurlCallbacks.kt

```diff
@@ -0,0 +1,72 @@
+package io.ktor.client.engine.curl
+
+import kotlinx.cinterop.*
+import libcurl.*
+import libcurl.CURLMsg.*
+import platform.posix.size_t
+import platform.posix.memcpy
```

Comment:
Please use `*` imports. You could configure it in kotlin codestyle section of IDEA